bitnami/elasticsearch/README.md

```diff
@@ -171,6 +171,37 @@ The following table lists the configurable parameters of the Elasticsearch chart
+| `curator.image.tag`                               | Elasticsearch Curator image tag                                                                                                                           | `{TAG_NAME}`                                            |
+| `curator.image.pullPolicy`                        | Elasticsearch Curator image pull policy                                                                                                                   | `{TAG_NAME}`                                            |
+| `curator.cronjob.schedule`                        | Schedule for the CronJob                                                                                                                                  | `0 1 * * *`                                             |
+| `curator.cronjob.annotations`                     | Annotations to add to the cronjob                                                                                                                         | {}                                                      |
+| `curator.cronjob.concurrencyPolicy`               | `Allow|Forbid|Replace` concurrent jobs                                                                                                                    | `nil`                                                   |
+| `curator.cronjob.failedJobsHistoryLimit`          | Specify the number of failed Jobs to keep                                                                                                                 | `nil`                                                   |
+| `curator.cronjob.successfulJobsHistoryLimit`      | Specify the number of completed Jobs to keep                                                                                                              | `nil`                                                   |
+| `curator.cronjob.jobRestartPolicy`                | Control the Job restartPolicy                                                                                                                             | `Never`                                                 |
+| `curator.podAnnotations`                          | Annotations to add to the pod                                                                                                                             | {}                                                      |
+| `curator.rbac.enabled`                            | Enable RBAC resources                                                                                                                                     | `false`                                                 |
+| `curator.serviceAccount.create`                   | Create a default serviceaccount for elasticsearch curator                                                                                                 | `true`                                                  |
```

Comment:
Shouldn't we use `curator.rbac.enabled` to decide whether creating the serviceAccount or not the serviceAccount when is created?

Reply:
I don't think so. You might want to use an existing service account. Also, this is similar to what we do in other charts (i.e. Metrics Server, NGINX Ingress Controller, Fluentd, MariaDB Galera...). I wouldn't change it.